gradle.properties

```diff
@@ -1,6 +1,6 @@
 # Package definitions
 GROUP=io.kategory
-VERSION_NAME=0.3.4-SNAPSHOT
+VERSION_NAME=0.3.5
```

Comment:
I'm working on a script that adds -SNAPSHOT to every non-release build, but you are right, until then it should stay 😄